src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,41 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { useContext } from 'react';
+import { PeopleContext } from '../../peopleContext';
+import { getPersonByName } from '../../utils/getPeopleByName';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug: userId, sex, born, died, fatherName, motherName } = person;
+  const { slug: selectedUserId } = useParams();
+
+  const people = useContext(PeopleContext);
+
+  const mother = getPersonByName(motherName, people);
+  const father = getPersonByName(fatherName, people);
+
+  return (
+    <tr
+      data-cy="person"
+      className={userId === selectedUserId ? 'has-background-warning' : ''}
+    >
+      <td>
+        <PersonLink person={person} />
+      </td>
```

Comment:
Code Quality Issue: The `PersonLink` component is used to render the person's name. However, it is not clear from the component's name that it is used for this purpose. Consider renaming the component to something more descriptive, like `PersonNameLink`.